### ISSUE TITLE
$extra data not work with times

### ISSUE THREAD

**Issue by godkinmo** [christophrumpel/laravel-factories-reloaded#47]
When override getDefaults data with times function, seem to be empty array in $extra variable

```
UserFactory::new()
    ->times(10)
    ->create(['age' => 30]);
```

**Comment by christophrumpel** [comment]
Thanks, I will take a look right now.

**Comment by christophrumpel** [comment]
I just fixed it and pushed a new release v1.0.6. Please update the version of the package and thanks a lot for telling me about it.
